[2024-04-13 23:06:00] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 23:06:05] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 23:06:10] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 23:06:15] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 23:06:20] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 23:06:22] [warning] server.go:631 p3Chain/p2p/server.(*Server).setupConn inbound conn 6c72c34988764114 [::1]:37390 failed checkpoint posthandshake: Connected to self
[2024-04-13 23:06:22] [warning] server.go:631 p3Chain/p2p/server.(*Server).setupConn static dial conn 6c72c34988764114 [::1]:20131 failed checkpoint posthandshake: Connected to self
[2024-04-13 23:06:25] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 23:06:30] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 23:06:35] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 23:06:40] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 23:06:45] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 23:06:50] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 23:06:52] [warning] server.go:631 p3Chain/p2p/server.(*Server).setupConn inbound conn 6c72c34988764114 [::1]:46540 failed checkpoint posthandshake: Connected to self
[2024-04-13 23:06:52] [warning] server.go:631 p3Chain/p2p/server.(*Server).setupConn static dial conn 6c72c34988764114 [::1]:20131 failed checkpoint posthandshake: Connected to self
[2024-04-13 23:06:55] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
